build.gradle

```diff
@@ -25,6 +25,11 @@ allprojects {
   if (!buildRoot.isEmpty()) {
     buildDir = buildRoot + project.path.replace(":", "/") + "/build"
   }
+
```

Comment:
Mark, can you wrap this in a whenReady closure?

```
gradle.taskGraph.whenReady({ graph -> 
   ...
}
```

That will avoid potential ordering problems...